docs/index.md

```diff
@@ -1239,20 +1254,13 @@ Options:
 
 The VMware vSphere driver uses the latest boot2docker image.
 
-#### exoscale
-Create machines on [exoscale](https://www.exoscale.ch/).
-
-Get your API key and API secret key from [API details](https://portal.exoscale.ch/account/api) and pass them to `machine create` with the `--exoscale-api-key` and `--exoscale-api-secret-key` options.
-
-Options:
-
- - `--exoscale-api-key`: Your API key.
- - `--exoscale-api-secret-key`: Your API secret key.
- - `--exoscale-instance-profile`: Instance profile. Default: `small`.
- - `--exoscale-disk-size`: Disk size for the host in GB. Default: `50`.
- - `--exoscale-security-group`: Security group. It will be created if it doesn't exist. Default: `docker-machine`.
-
-If a custom security group is provided, you need to ensure that you allow TCP port 2376 in an ingress rule.
+## Base Operating Systems
+The default base operating system for Machine is Boot2Docker on local providers
+(VirtualBox, Fusion, Hyper-V, etc) and the latest Ubuntu LTS supported
+by the cloud provider.  RedHat Enterprise Linux is also supported.  To use
```

Comment:
Oh, I guess because of the RHN stuff? It'd probably be a good idea to get CentOS support in here before too long, but maybe as a follow-up, not necessarily this PR.